js/dashboard.js

```diff
@@ -26,35 +28,107 @@ $(function() {
             "lat": 45.281041,
             "lng": -123.061896
         },
+        'nodeLoc': [
+            [45.281679, -123.062533, 58],
+            [45.281554, -123.061838, 17],
+            [45.281423, -123.061124, 55],
+            [45.281249, -123.062615, 25],
+            [45.281123, -123.061897, 42],
+            [45.280989, -123.061145, 51],
+            [45.280800, -123.062877, 17],
+            [45.280680, -123.062076, 21],
+            [45.280511, -123.061333, 66]
+        ],
+        // Boundary coordinates for vineyard
+        // Must be listed in drawing order
+        'propertyBoundary': [
+            [45.282053, -123.062833],
+            [45.281605, -123.060469],
+            [45.280016, -123.061092],
+            [45.280269, -123.061468],
+            [45.280390, -123.062234],
+            [45.280377, -123.063102],
+            [45.280464, -123.063369]
+        ]
     });
-})
+});
 
 ////////////////////////////////// Map code ///////////////////////////////////
 var map, mapsReady = false, mapsData = null;
 function createMap(data) {
+    // Calculate map bounds
+    var latOffset = .00145; // perhaps calculate these offsets based
+    var lngOffset = .00525; // on the surface area of the vineyard
+    var southWest = {'lat': data.center.lat - latOffset, 'lng': data.center.lng - lngOffset};
+    var northEast = {'lat': data.center.lat + latOffset, 'lng': data.center.lng + lngOffset};
+
+    // Initialize map
     mapsData = data;
-    map = new google.maps.Map($(".map")[0], {
+    map = L.map('map', {
         center: data.center,
+        doubleClickZoom: false,
+        layers: MQ.satelliteLayer(),
+        maxBounds: [southWest, northEast],
+        minZoom: 18,
```

Comment:
This zoom level is too high for small screens - it cuts off the edges of the field in mobile view.